client/reader/_style.scss

```diff
@@ -23,7 +23,6 @@
 .is-reader-page .follow-button {
 	border: 0;
 	border-radius: 0;
-	float: right;
 }
```

Comment:
Done in: https://github.com/Automattic/wp-calypso/pull/14881/files#diff-418e0c0a76dea69ba95b1efe94eaf91eR11